# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/MediaEngine.h
# Commit: 1248d2f659a9
# Full Hash: 1248d2f659a9f341f0de5d6558eecf364fc6f0a2
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2019-03-29 21:53:01
# Regressor Bug: 1494675
# File Overlap Count: 1
# Description:
#   Bug 1494675 - Remove windowID to MediaEngineSource mappings from MediaEngines. r=padenot
#   
#   These once served a caching-and-reuse purpose it seems, but it makes less sense
#   when sources are not shared.
#   
# ==============================================================================

diff -r 20543d03a4cd -r 1248d2f659a9 dom/media/webrtc/MediaEngine.h
--- a/dom/media/webrtc/MediaEngine.h	Wed Mar 27 14:10:24 2019 +0000
+++ b/dom/media/webrtc/MediaEngine.h	Wed Mar 27 14:10:10 2019 +0000
@@ -45,7 +45,6 @@
                                 MediaSinkEnum,
                                 nsTArray<RefPtr<MediaDevice>>*) = 0;
 
-  virtual void ReleaseResourcesForWindow(uint64_t aWindowId) = 0;
   virtual void Shutdown() = 0;
 
   virtual void SetFakeDeviceChangeEvents() {}